NEWS.md

```diff
@@ -1,3 +1,7 @@
+### vimpact 0.0.4
+https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4444
+Fix extract_vaccine_history() and cohort_deaths_all_cause().
```

Comment:
```suggestion

* Fix extract_vaccine_history() and cohort_deaths_all_cause(). (VIMC-4444)
```

this is how we normally do news items if they reference issues. You can merge this suggestion directly from this comment (see buttons above)